google/cloud/spanner/client_test.cc

```diff
@@ -1056,8 +1057,8 @@ TEST(ClientTest, ProfileQueryWithOptionsSuccess) {
   };
   int row_number = 0;
   for (auto& row : StreamOf<RowType>(rows)) {
-    EXPECT_STATUS_OK(row);
-    EXPECT_EQ(*row, expected[row_number]);
+    ASSERT_LT(row_number, expected.size());
```

Comment:
Another opportunity for `ElementsAre(IsOkAndHolds(...`

Reply:
Done.